### PR TITLE
fix(e2e) - Fix firefox e2e tests

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/spec/content_spec.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ describe('Content', function() {
 
     beforeEach(function() {
         openUrl('/#/workspace/content')();
-        workspace.openPersonal();
-        expect(element.all(by.repeater('items._items')).count()).toBe(3);
+        workspace.switchToDesk('PERSONAL');
+        expect(element.all(by.repeater('items._items')).count()).toBe(6);
```
Comment (petrjasek) [inline client/spec/content_spec.js]: fyi you can use `.toBeGreaterThan(x)` so we don't need to update specs when there is a new item added

**Comment by petrjasek** [comment]
can you push it also to `fix-e2e-setup` branch so we can test it on bamboo before merging to master?

**Comment by mugurrus** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**